### PR TITLE
Remove flag_file_name from all zone configs

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

The flag_file_name is no loger used but still included in almost all zone configs.

## Description

This removes the flag_file_name from all the zone configs and updates any typings to account for this.

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: scripts/remove_flag_file_name.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import yaml
```
Comment (madsnedergaard) [inline scripts/remove_flag_file_name.py:1]: I assume this made things much easier when doing it :D
Not sure we really need to keep the script though?

Comment (VIKTORVAV99) [reply scripts/remove_flag_file_name.py:1]: Considering there was 250+ files to look though it sure did 😅

But you are right, we don't need to keep the script. I'll push a commit that removes it.

**Comment by VIKTORVAV99** [comment]
Hmm Cypress e2e is failing because of some unknown script error?
This should not have affected that in any way 😅

**Comment by VIKTORVAV99** [comment]
Looks like cypress is failing on master as well so it's not related to this PR.

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**